### DISCUSSION TITLE
Could opencontrol have a "red box" in session view?

### DISCUSSION THREAD

**Discussion by ll-dasein-ll** [KBLiveSolutions/open.control#24] [Feature Requests & Ideas]
This might sound like a strange request, but is there any way red box functionality could be added to the opencontrol, even if it only creates a ring that surrounds a single cell in session view?

I'm asking because by using a free Max For Live device called Session Ring Control (https://maxforlive.com/library/device/3525/session-ring-control), I think I could link the red boxes of the opencontrol and my Akai APC 24 keyboard together and find my way around my larger sets more easily, thanks to the coloured LEDs on the opencontrol being able to display the colour of the cells and the screen displaying the name of the focused track.  The ACP 24 only has green, red and yellow lights and no text screen so it's easy to get lost in large sets with lots of tracks.

**Comment by KBLiveSolutions** [comment]
This is already possible: in the editor, check the "create session box" in the top menu.

**Reply by ll-dasein-ll** [reply]
Oh wow, I hadn't noticed that - awesome!  I'll try this out later.  Thanks for the tip!
